tests/test_bot.py

```diff
@@ -286,6 +286,7 @@ def test_tech_support_listener(mock_app, text, channel, event_kwargs, respost_ex
         assert recorder.mock_received_requests_kwargs["/chat.postMessage"][0] == {
             "text": "http://test",
             "channel": "C0001",
+            "team_id": "T111",
```

Comment:
So I guess slackbolt's `say` is now adding the team into the context it passes along to `postMessage`; should we maybe be asserting just the things we know we pass, rather than the exact kwargs?  Although, it's probably not a bad thing that the test fail and alert us to changes